tools/shared.py

```diff
@@ -2555,19 +2555,29 @@ def add_to_path(dirname):
       logger.debug('closure compiler: ' + ' '.join(cmd))
 
       proc = run_process(cmd, stderr=PIPE, check=False, env=env)
```

Comment:
I think the stylized output (yellow and red printing to console) is nice to have, so that the warnings and errors stand out better than if the output was regular/gray.

Reply:
My understanding is that that way to achieve the color output would be to remove the `stderr=PIPE` and allow the closure output to be connected to the tty, right?  It sounds like you agree but I don't see that in this version of the code.

Or maybe closure doesn't colorize its own output?

In any case the `closure compiler completed with warnings` would still stand out in red below the output right?

Reply:
Yeah, Closure does not colorize its output.